### ISSUE TITLE
Pretraining issue

### ISSUE THREAD

**Issue by naot97** [zengyan-97/CCLM#18]
Hi, 

I'm retraining pre-training step with some changes, but my loss result still wasn't reduce in second epoch. I wonder mistakes is my code or not. Can you share the chart or log of you pre training. Thanks.
